demos/try10_test_cases.py

```diff
@@ -12,7 +12,7 @@
 model_deconv, data_deconv, problem_info_deconv = cuqi.testproblem.Deconvolution1D.get_components()
 
 #%% Deblur Test problem 
-model_deblur, data_deblur, problem_info_deblur = cuqi.testproblem.Deblur.get_components()
+model_deblur, data_deblur, problem_info_deblur = cuqi.testproblem._Deblur.get_components()
```

Comment:
An issue can be added: Update demos/try scripts that uses _Deblur to use Deconvolution1D instead